arches/install/arches-project

```diff
@@ -1,184 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Without changing the entrypoint/name of the command, I renamed this file so that setuptools could find it -- it only likes scripts that have python-importable names, so `arches-project` in the filepath needed to become `arches_project` (again, without any impact on the actual name of the CLI command, preserved in pyproject.toml)